fedbiomed/node/cli.py

```diff
@@ -459,7 +461,7 @@ def start(self, args):
         p = Process(
```

Comment:
`start.add_argument(
            "--gpu-num",
            "-gn",
            type=int,
            nargs="?",
            required=False,
            default=1,
            help="Number of GPU that is going to be used")
`

I think default value should be `default=0`

Reply:
otherwise i got a warning about not finding the correct gpu (using only `--gpu-only` flag)

Reply:
Hi @ybouilla,
I didn't understand the issue. Was the default was `1` all the time, or `0`. You used to get similar warnings before? and does node uses gpu if `gpu-only` is activated and `gpu` count is 0?